.github/workflows/docs/intro.txt

```diff
@@ -6,10 +6,123 @@ pytket-extensions
    :align: right
 
 
-These extensions enable `CQC`_ `pytket`_ to be used in conjunction with other
-platforms. Each extension adds either new methods to the ``pytket`` package to
-convert between circuit representations, or new backends to which ``pytket``
-circuits can be submitted.
+These extensions enable ``pytket`` to interface with different hardware platforms and classical simulators. One can also access additional quantum devices and simulators via the cloud through the extensions for `Azure <https://cqcl.github.io/pytket-extensions/api/qsharp/api.html#pytket.extensions.qsharp.AzureBackend>`_ and `Braket <https://cqcl.github.io/pytket-extensions/api/braket/api.html#pytket.extensions.braket.BraketBackend>`_ . 
+In pytket a ``Backend`` represents a connection to a QPU (Quantum Processing Unit) or simulator for processing quantum circuits. The types of ``Backend`` available in pytket are the following.
+
+Types of Backend
+================
+
+* **QPUs** - These are real quantum computers that return shots based results. E.g the `QuantinuumBackend <https://cqcl.github.io/pytket-quantinuum/api/>`_ .
```

Comment:
The right link should be cqcl.github.io/pytket-quantinuum/api/api.html#pytket.extensions.quantinuum.QuantinuumBackend

Reply:
Thanks
Should be correct now.